Assets/HoloToolkit/SpatialMapping/Scripts/ObjectSurfaceObserver.cs

```diff
@@ -15,7 +15,11 @@ public class ObjectSurfaceObserver : SpatialMappingSource
 
         private void Start()
         {
-            if (!UnityEngine.XR.XRDevice.isPresent && Application.isEditor)
+            if (
+#if UNITY_2017_2_OR_NEWER
+                !UnityEngine.XR.XRDevice.isPresent && 
```

Comment:
If we really need to support all these previous versions, you should probably add `!UnityEngine.VR.VRDevice.isPresent` for non-2017.2.

Reply:
@keveleigh  These are outdated.  I'm pretty sure I updated this to what you're suggesting, but I'll check it out again.